compliance-monitor/monitor.py

```diff
@@ -331,8 +328,8 @@ def get_scopes():
             _scopes[ident] = current = {'_counter': -1}
     if current['_counter'] != counter:
         current.clear()
-        import_cert_yaml_dir(yaml_path, current)
         current['_counter'] = counter
+        import_cert_yaml_dir(yaml_path, current)
```

Comment:
@mbuechse An (unrelated?) fix sneaked in here, please double check if that's wanted. But it looks like a valid bugfix to me.

Reply:
Yes, that bugfix is intended. It sneaked in here like you say, and I didn't want to spend the time to untangle it.